systers_portal/meetup/views.py

```diff
@@ -224,6 +225,20 @@ def get_context_data(self, **kwargs):
         context['coming_no'] = len(coming_list) + len(plus_one_list)
         context['not_coming_no'] = len(not_coming_list)
         context['share_message'] = self.object.title + " @systers_org "
+        meetups =\
+            Meetup.objects.filter(~Q(slug=self.object.slug)).filter(date__gte=datetime.date.today())
+        list_scores = []
+        if meetups:
+            for meetup in meetups:
+                val = compare(self.object.description, meetup.description)
+                list_scores.append({'id': meetup.id, 'score': val})
+        list_scores.sort(key=operator.itemgetter('score'), reverse=True)
```

Comment:
Why not just put everything from here under
```
if meetups:
```

Reply:
Done